doc/user/layouts/partials/create-source/connector/kafka/with-options.html

```diff
@@ -27,12 +28,14 @@
 
 Field | Value | Description
 ------|-------|------------
-`sasl_kerberos_keytab` | `text` | The absolute path to your keytab.
-`sasl_kerberos_kinit_cmd` | `text` | Shell command to refresh or acquire the client's Kerberos ticket.
-`sasl_kerberos_min_time_before_relogin` | `text` | Minimum time in milliseconds between key refresh attempts. Disable automatic key refresh by setting this property to 0.
-`sasl_kerberos_principal` | `text` | Materialize Kerberos principal name. Required for `sasl_plaintext`.
-`sasl_kerberos_service_name` | `text` | Kafka's service name on its host, i.e. the service principal name not including `/hostname@REALM`.
-`sasl_mechanisms` | `text` | The SASL mechanism to use for authentication. Currently, the only supported mechanisms are `'GSSAPI'` (the default) and `'PLAIN'`.
+`sasl_mechanisms` | `text` | The SASL mechanism to use for authentication. Currently, the only supported mechanisms are `GSSAPI` (the default) and `PLAIN`.
+`sasl_username` | `text` | Username required if `sasl_mechanisms` is `PLAIN`.
```

Comment:
suggestion (feel free to reject/ignore): 

1. add parentheses:
> Username (required if sasl_mechanisms is PLAIN).
2. or drop `Username` altogether:
> (required if sasl_mechanisms is PLAIN)

I find that the "Username" and "Password is somewhat unnecessary, and the lack of punctuation between "Username" and "requirement" slightly hinders readability